<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMTPClient Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>-1</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBoxCredentials">
      <property name="title">
       <string>From</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="horizontalSpacing">
          <number>9</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEditHost">
           <property name="text">
            <string>smtp.gmail.com</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelEmailCredentials">
           <property name="text">
            <string>E-mail</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QSpinBox" name="spinBoxPort">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>465</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="lineEditPasswordCredentials">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditEmailCredentials"/>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="labelHost">
           <property name="text">
            <string>Host</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="labelPasswordCredentials">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="labelPort">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxFrom">
      <property name="title">
       <string>From</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelEmailFrom">
           <property name="text">
            <string>E-mail</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditEmailFrom"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxTo">
      <property name="title">
       <string>To</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="labelEmail">
           <property name="text">
            <string>E-mail</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditEmailTo"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxMessage">
      <property name="title">
       <string>Message</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="labelSubject">
           <property name="text">
            <string>Subject</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditSubject"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="labelContent">
           <property name="text">
            <string>Content</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditContent"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonClear">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSend">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditEmailCredentials</tabstop>
  <tabstop>lineEditPasswordCredentials</tabstop>
  <tabstop>lineEditHost</tabstop>
  <tabstop>spinBoxPort</tabstop>
  <tabstop>lineEditEmailFrom</tabstop>
  <tabstop>lineEditEmailTo</tabstop>
  <tabstop>lineEditSubject</tabstop>
  <tabstop>textEditContent</tabstop>
  <tabstop>pushButtonSend</tabstop>
  <tabstop>pushButtonClear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
